pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -39,6 +40,59 @@ func createSquashDictInnerAssertLenKeysHinter() (hinter.Hinter, error) {
 	return newSquashDictInnerAssertLenKeysHint(), nil
 }
 
+// SquashDictInnerContinueLoop hint determines if the loop should continue
+// based on remaining access indices
+//
+// `newSquashDictInnerContinueLoopHint` takes 1 operander as argument
+//   - `loopTemps` variable is a struct containing a `should_continue` field
+//
+// `newSquashDictInnerContinueLoopHint`writes 0 or 1 in the `should_continue` field
+// depending on whether the `current_access_indices` array contains items or not
+func newSquashDictInnerContinueLoopHint(loopTemps hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "SquashDictInnerContinueLoop",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> ids.loop_temps.should_continue = 1 if current_access_indices else 0
+
+			currentAccessIndices_, err := ctx.ScopeManager.GetVariableValue("current_access_indices")
+			if err != nil {
+				return err
+			}
+
+			currentAccessIndices := currentAccessIndices_.([]fp.Element)
+
+			loopTempsAddr, err := loopTemps.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			resultAddr, err := loopTempsAddr.AddOffset(4)
+			if err != nil {
+				return err
+			}
+
+			resultMemZero := memory.MemoryValueFromFieldElement(&utils.FeltZero)
+			resultMemOne := memory.MemoryValueFromFieldElement(&utils.FeltOne)
```

Comment:
Why not add in the corresponding `if` statement?

Reply:
yeah true, my bad